### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Naveen481**
1. Removed RabbitMQ dependency.
2. Tested the CSD.
3. Configurations updated to broker_url. 
4. Updated Readme

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 target/
 *.swp
+.gitignore
+*.DS_Store
+.idea/*
```
Comment (rssanders3) [inline .gitignore]: Add *.iml

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rssanders3) [inline .idea/vcs.xml]: Remove the .idea directory. This is specific to your development environment.

File: README.md [hunk 3]
```diff
@@ -5,17 +5,30 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 ## Requirements
 - A supported operating system.
 - MySQL or PostgreSQL database in which to store Airflow metadata.
+<<<<<<< HEAD
```
Comment (rssanders3) [inline README.md]: Looks like some merge issues. Please resolve these.

File: README.md [hunk 4]
```diff
@@ -5,17 +5,30 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
...
+- RabbitMQ
+- [Airflow Parcel](https://github.com/teamclairvoyant/apache-airflow-parcels)
+
+### Currently Supported Versions of Airflow
+- Airflow 1.9.0
+- Airflow 1.10.0
```
Comment (rssanders3) [inline README.md]: Be sure to add the specific sub-versions that are supported.

File: README.md [hunk 5]
```diff
@@ -5,17 +5,30 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
...
 - Airflow 1.10
+>>>>>>> 11b39ebe4b384d6ad1c1735474123e5af2c8676a
 
 ### Currently Supported Operating Systems
 - CentOS/RHEL 6 & 7
 - Debian 8
 - Ubuntu 14.04, 16.04, & 18.04
 
 ## Installing the CSD
+<<<<<<< HEAD
```
Comment (rssanders3) [inline README.md]: More Merge Issues here, please resolve.

File: README.md [hunk 6]
```diff
@@ -69,9 +92,13 @@ Webserver: Airflow Webserver role runs the Airflow Web UI. Webserver role can be
 
 Scheduler: Airflow Scheduler role is used to schedule the Airflow jobs. This is limited to one instance to reduce the risk of duplicate jobs.
 
+<<<<<<< HEAD
```
Comment (rssanders3) [inline README.md]: More Merge Issues here, please resolve.

File: README.md [hunk 7]
```diff
@@ -128,23 +155,30 @@ make dist
 Update the `version` file before running `make dist` if creating a new release.
 
 ## Limitations:
+<<<<<<< HEAD
```
Comment (rssanders3) [inline README.md]: More Merge Issues here, please resolve.

File: src/scripts/stop_airflow_scheduler.sh [hunk 8]
```diff
@@ -1,2 +1,5 @@
+<<<<<<< HEAD
```
Comment (rssanders3) [inline src/scripts/stop_airflow_scheduler.sh]: More Merge Issues here, please resolve.

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Naveen481) [reply README.md]: We built the CSD's for Airflow 1.9.0 and Airflow 1.10.0. Do you want to include all the subversions like Airflow 1.10.1b1?

Comment (rssanders3) [reply README.md]: Based on what's in the archive.clairvoyantsoft.com bucket, I see that Airflow 1.10.3 is available. Not 1.10.0 or 1.10.1b1.

File: src/scripts/stop_airflow_scheduler.sh (hunk 8 above)
Comment (rssanders3) [reply src/scripts/stop_airflow_scheduler.sh]: Merge Conflict still here

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Naveen481) [reply README.md]: Great. Thanks

**Pull request merged**